src/targets/gpu/include/migraphx/gpu/convolution.hpp

```diff
@@ -163,6 +167,20 @@ struct miopen_convolution
         auto w_desc                = make_tensor(reshape_if_1d(inputs[1]), int8_x4_format);
         auto y_desc                = make_tensor(reshape_if_1d(output_shape));
         std::size_t workspace_size = 0;
+
+        // TODO:  Using an environment variable to disable convolution tuning while still allowing
+        // other (i.e. GEMM) tuning with --exhaustive-tune, is a workaround for a problem in which
+        // convolution tuning takes multiple hours.
+        const bool convo_tune_enabled = enabled(MIGRAPHX_ENABLE_CONV_TUNING{});
+        if(ctx.get_exhaustive_tune_flag() and not convo_tune_enabled)
+        {
```

Comment:
This causes a flood of warning messages.  Do we have a tidy way to limit this message to only output once per compilation?